src/main/java/com/qunite/api/web/controller/QueueController.java

```diff
@@ -70,9 +73,46 @@ public ResponseEntity<Integer> membersAmount(@PathVariable Long id) {
       @ApiResponse(responseCode = "200"),
       @ApiResponse(responseCode = "404", content = @Content())
   })
-  public ResponseEntity<Integer> memberPosition(@PathVariable(value = "id") Long queueId,
+  public ResponseEntity<Integer> memberPosition(@PathVariable Long id,
                                                 @PathVariable(value = "member-id") Long memberId) {
-    return ResponseEntity.of(queueService.getMemberPositionInQueue(memberId, queueId));
+    return ResponseEntity.of(queueService.getMemberPositionInQueue(memberId, id));
+  }
+
+  @PostMapping("/{id}/members")
+  @Operation(summary = "Enroll member to queue", responses = {
+      @ApiResponse(responseCode = "200"),
+      @ApiResponse(responseCode = "404", content = @Content())
+  })
+  public ResponseEntity<Void> enrollMemberToQueue(@PathVariable Long id,
+                                                  Principal principal) {
+    queueService.enrollMemberToQueue(principal.getName(), id);
+    return ResponseEntity.ok().build();
```

Comment:
When entering a queue, we can return a place in the queue

Reply:
I've created an issue for the refactor of this endpoint:
https://github.com/q-unite/qunite-api/issues/90